packages/marko-web-native-x/utils/convert-story-to-content.js

```diff
@@ -0,0 +1,46 @@
+const { get, getAsObject } = require('@parameter1/base-cms-object-path');
+
+module.exports = (story = {}, { sectionName = 'Sponsored' } = {}) => {
+  const advertiser = getAsObject(story, 'advertiser');
+  const contentImage = getAsObject(story, 'primaryImage');
+  const advertiserImage = getAsObject(story, 'advertiser.logo');
+  const startDate = get(story, 'publishedAt');
+  return {
+    id: story.id,
+    name: story.title,
+    shortName: story.title,
+    body: story.body,
+    typeTitled: 'Text Ad',
```

Comment:
The default type should be an Article, not a Text Ad...

Can you double-check all of these to make sure they work with a Story, since this was copied-pasted from the campaign builder? :)